repos/spack_repo/builtin/packages/palace/package.py

```diff
@@ -16,10 +16,12 @@ class Palace(CMakePackage, CudaPackage, ROCmPackage):
 
     homepage = "https://github.com/awslabs/palace"
     git = "https://github.com/awslabs/palace.git"
+    license("Apache-2.0")
 
-    maintainers("hughcars", "simlap", "cameronrutherford")
+    maintainers("hughcars", "simlap", "cameronrutherford", "gbozzola")
```

Comment:
```suggestion
    maintainers("hughcars", "simlap", "cameronrutherford", "sbozzolo")
```

I think this has to be the GitHub handle, so that we get notified

Reply:
Yes I think this was a bug I introduced in PR review. We want the GitHub handle.